model/minio/minio_server.rb

```diff
@@ -26,7 +26,7 @@ def hostname
   end
 
   def private_ipv4_address
```

Comment:
It seems this helper is used in a few places and was added because the old method was lengthy. Since the helper in the VM makes things easier, should we remove the MinIO helper and just use `server.vm.private_ipv4`?

Reply:
I agree it's used in few places but I prefer to keep it because mentally this makes it easier for me to read the code on these few places.